samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js

```diff
@@ -172,30 +202,51 @@ const StorybookDecorator = (story, config) => {
 			highlight: '#2a48ca'
 		}
 	};
-	const skinFromURL = getPropFromURL('skin');
-	const currentSkin = skinFromURL ? skinFromURL : Config.defaultProps.skin;
-	const newSkin = (memory.skin !== currentSkin);
-	memory.skin = currentSkin;  // Remember the skin for the next time we load.
-	const accentFromURL = getPropFromURL('accent');
-	const highlightFromURL = getPropFromURL('highlight');
-	const localeFromURL = getPropFromURL('locale');
-	const currentLocale = localeFromURL ? localeFromURL : Config.defaultProps.locale;
-	const locale = select('locale', locales, Config, currentLocale);
-	const allSkins = boolean('show all skins', Config);
+
+	if (config.parameters) {
+		if (config.parameters.info && config.parameters.info.text) {
+			config.description = config.parameters.info.text;
+		}
+		if (config.parameters.props) {
+			config.props = config.parameters.props;
+		}
+	}
+
+	const args = getArgs();
+	const classes = {
+		aria: boolean('debug aria', DevelopmentConfig, getKnobFromArgs(args, 'debug aria')),
+		layout: boolean('debug layout', DevelopmentConfig, getKnobFromArgs(args, 'debug layout')),
+		spotlight: boolean('debug spotlight', DevelopmentConfig, getKnobFromArgs(args, 'debug spotlight'))
+	};
+	if (Object.keys(classes).length > 0) {
+		classes.debug = true;
+	}
 	const skinKnobs = {};
+	const allSkins = boolean('show all skins', Config);
+
 	if (!allSkins) {
-		skinKnobs.skin = select('skin', skins, Config, currentSkin);
+		skinKnobs.skin = select('skin', skins, Config, getKnobFromArgs(args, 'skin'));
+		const useSkinDefaultStyles = boolean('default skin styles', Config);
```

Comment:
How about the name defaultSkinStyle instead of useSkinDefaultStyles?  It seems to me like hook.